src/modes/OptimInterface.jl

```diff
@@ -50,14 +50,23 @@ function Base.show(io::IO, m::ModeResult)
     show(io, m.values.array)
 end
 
-function StatsBase.coeftable(m::ModeResult)
+function StatsBase.coeftable(m::ModeResult; level::Real=0.95)
     # Get columns for coeftable.
-    terms = StatsBase.coefnames(m)
-    estimates = m.values.array[:,1]
+    terms = String.(StatsBase.coefnames(m))
```

Comment:
IIRC it is more common to write
```suggestion
    terms = string.(StatsBase.coefnames(m))
```